test_to_do_app/test_to_do_app.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime,timedelta
```

Comment:
Not directly related to this file, but it would be good practice to add an empty `__init__.py` file to this directory; different Python versions are inconsistent on exactly when they need one, so even when it's not strictly necessary it's usually worthwhile adding one for simplicity (and this might save you headaches later when we try to migrate the tests inside Docker!)